src/shared/components/Header/components/PageHeader/PageHeader.jsx

```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
-import Button from '../../../../shared/components/Button/Button';
+import Button from '../../../Button/Button';
 import Sidebar from '../Sidebar/Sidebar';
-import './Header.scss';
-import menu from '../../../../shared/images/header/menu.svg';
-import menu2 from '../../../../shared/images/header/menu2.png';
+import './styles/PageHeader.scss';
```

Comment:
Please move styles import to be the last import. It's not an issue, but good for readability.